src/plugins/controls/public/react_controls/controls/data_controls/range_slider/get_range_slider_control_factory.tsx

```diff
@@ -60,7 +61,6 @@ export const getRangesliderControlFactory = (
       const controlFetch$ = controlGroupApi.controlFetch$(uuid);
       const loadingMinMax$ = new BehaviorSubject<boolean>(false);
       const loadingHasNoResults$ = new BehaviorSubject<boolean>(false);
-      const dataLoading$ = new BehaviorSubject<boolean | undefined>(undefined);
```

Comment:
I am not sure if we want to remove `dataLoading$`. `dataLoading$` from data control is tracking `dataView` loading. I think instead, we should keep this observable and instead update `dataLoadingSubscription` to also listen to `dataControl.api.dataLoading`

Reply:
Ahh yeah, that's a good control - that could definitely cause conflicts. Did this for the range slider control + also migrated the options list control to do the same thing in 3cc39f7a1ba21b9ff095ff97b5ca417d8012d209